pr-data.csv

```diff
@@ -4339,6 +4339,7 @@ https://github.com/Ericsson/ecchronos,5d58b0e4c922822c9df3d893fff2246d5784b11f,c
 https://github.com/Ericsson/ecchronos,5d58b0e4c922822c9df3d893fff2246d5784b11f,core,com.ericsson.bss.cassandra.ecchronos.core.repair.TestRepairTask.testPartialRepair,ID,Accepted,https://github.com/Ericsson/ecchronos/pull/244,
 https://github.com/Ericsson/ecchronos,5d58b0e4c922822c9df3d893fff2246d5784b11f,core,com.ericsson.bss.cassandra.ecchronos.core.repair.TestRepairTask.testRepairHalf,ID,Accepted,https://github.com/Ericsson/ecchronos/pull/244,
 https://github.com/Ericsson/ecchronos,5d58b0e4c922822c9df3d893fff2246d5784b11f,core,com.ericsson.bss.cassandra.ecchronos.core.repair.TestRepairTask.testRepairSuccessfully,ID,Accepted,https://github.com/Ericsson/ecchronos/pull/244,
+https://github.com/Ericsson/ecchronos,3e0a36ea4ead09c5a9633e29f4c02f6b58ebc25d,core,com.ericsson.bss.cassandra.ecchronos.core.TestLockCache.testEqualsContract,ID,,,https://github.com/jqno/equalsverifier/pull/1161
```

Comment:
Please change the note to "fixed in another repo https://github.com/jqno/equalsverifier/pull/1161". Otherwise, it's not clear what the PR is for. Thanks.